admin-ui/plugins/auth-server/components/AuthN/Basic.js

```diff
@@ -138,39 +87,26 @@ function AuthNListPage({ isBuiltIn = false }) {
           Container: (props) => <Paper {...props} elevation={0} />,
         }}
         columns={[
-          { title: `${t('fields.acr')}`, field: 'acrName' },
-          { title: `${t('fields.saml_acr')}`, field: 'samlACR' },
-          { title: `${t('fields.level')}`, field: 'level' },
+          { title: `acr`, field: 'acrName' },
+          { title: `saml acr`, field: 'samlACR' },
+          { title: `level`, field: 'level' },
           {
-            title: `${t('options.default')}`,
+            title: `default`,
             field: '',
-            render: (rowData) => {
-              return rowData.acrName === acrs.defaultAcr ? (
-                <i
-                  className="fa fa-check"
-                  style={{ color: customColors.logo, fontSize: '24px' }}
-                ></i>
-              ) : (
-                <i
-                  className="fa fa-close"
-                  style={{ color: customColors.accentRed, fontSize: '24px' }}
-                ></i>
-              )
-            },
+            render: (rowData) => (
+              <i
+                className={rowData.acrName === acrs.defaultAcr ? 'fa fa-check' : 'fa fa-close'}
+                style={{ color: customColors.logo, fontSize: '24px' }}
+              />
+            ),
           },
```

Comment:
Hard-coding strings for table column titles is bad because it violates internationalization best practices by embedding display text directly in the source code, which bypasses the localization framework and prevents the labels from being translated for different locales.